packages/Heading/src/Heading.styles.js

```diff
@@ -1,7 +1,7 @@
 import tokens from "@paprika/tokens";
 import stylers from "@paprika/stylers";
 
-const headingStyles = () => `
+const defaultHeadingStyles = `
```

Comment:
While you're at it, can you replace:
`line-height: 1.2` with `${stylers.lineHeight(-1)}`, please?

(Line 10 below)

Reply:
done